### ISSUE TITLE
Hitomi.la Search Bar Doesn't Work

### ISSUE THREAD

**Issue by BinariGuy** [avluis/Hentoid#262]
**Hentoid Version**: 1.5.11

**Device/Android Version**: M4 SS4452/5.1.1

**Issue details / Repro steps**: Searching in Hitomi.la bar doesn't show any results

**Stack trace / LogCat**:
```ruby
paste stack trace and/or log here
```

@AVnetWS/admin-team

**Comment by herdidi47** [comment]
Hi BinariGuy!

your issue is related to hitomi.la website instead of hentoid. else if you can search using normal browser but can't using hentoid webview then it was hentoid problem, but i just tried searching using hentoid and it works fine. maybe some screenshot/recording regarding that problem might help.

thanks

**Comment by RobbWatershed** [comment]
Heya!
In addition, could you tell us which version of Hentoid you're using, and what Android version your phone has?
Thanks for the report!

**Comment by RobbWatershed** [comment]
I'm pretty sure I have reproduced that issue on Android 5.1.1

I'll keep you updated for a solution

**Comment by BinariGuy** [comment]
Sorry but im unable to take a screenshot inapp

**Comment by RobbWatershed** [comment]
**TL;DR** This issue is directly linked to your hitomi.la and your Android version. We can't fix that.


**DETAILS**
Hitomi.la uses advanced JS syntax ([that one](https://caniuse.com/#feat=const) for instance) that is not recognized by the default system browser of older Android versions.

As a proof of what I'm saying, try going on hitomi and use search with your system browser (e.g. the "internet" app that shipped with your phone) and you'll see the result is the same as in Hentoid.


**WORKAROUNDS**
Since Hentoid embeds the Android default system browser, it means you have only two options
1/ upgrade your OS.
2/ browse & search hitomi with another browser app such as Chrome and use the "open with Hentoid" option to switch to Hentoid.
